src/popup/App.vue

```diff
@@ -1,56 +1,62 @@
 <template>
-  <div
-    class="app-wrapper"
-    :class="{
-      'show-header': showHeader,
-      'is-desktop-web': IS_WEB && !IS_MOBILE_DEVICE,
-      'is-extension': IS_EXTENSION,
-    }"
-  >
-    <button
-      v-if="qrScannerOpen"
-      class="camera-close-button"
+  <IonApp class="ionic-wrapper">
```

Comment:
We have this rule that says the class added to the root element should match the component name. So in this case it should be `app`.